cve_bin_tool/linkify.py

```diff
@@ -0,0 +1,15 @@
+from rich.style import Style
+from rich.text import Text, TextType
+
+RE_CVE = r"CVE\-\d{4}\-\d+"
```

Comment:
```suggestion
RE_CVE = r"CVE\-\d{4,7}\-\d+"
```

The old CVEs were always 4 digits, but since last year or so they can be longer.  I've seen them up to 7 characters.  (e.g. https://www.cvedetails.com/cve/CVE-2019-1020018/)

Reply:
As far as I know, I don't think this pattern: `r"CVE\-\d{4}\-\d+"` will create any problem. because first  4 digit after CVE always represent year and which have been 4 digit long since beginning which will match the pattern `d{4}` and last digits can be any unique number for that CVE so it can be longer than 4 digits which will match the pattern `d+`

Reply:
Ah, you're right, the year will still always be 4, that should have been 
```suggestion
RE_CVE = r"CVE\-\d{4}\-\d{4,7}"
```
But you're right, just `+` works just fine.  Apparently I shouldn't have given up coffee today, sorry!